addons/website/static/src/snippets/s_facebook_page/options.js

```diff
@@ -1,6 +1,8 @@
 odoo.define('website.s_facebook_page_options', function (require) {
 'use strict';
 
+const core = require('web.core');
+const _t = core._t;
 const options = require('web_editor.snippets.options');
```

Comment:
Minor comment: the convention is to do all the `require()` before assigning `_t`, so move `const _t = core._t;` 1 or 2 lines down :)